src/v/cluster/metrics_reporter.cc

```diff
@@ -271,6 +271,16 @@ metrics_reporter::build_metrics_snapshot() {
 
         snapshot.topic_count++;
         snapshot.partition_count += md.get_configuration().partition_count;
+        switch (md.get_configuration().properties.iceberg_mode) {
+        case model::iceberg_mode::disabled:
```

Comment:
is disabled state reported? i guess enabled may be inferred

Reply:
Yeah it will be inferred. Since this is the default value I don't think it's interesting either.